packages/web/utils/menuLinks.ts

```diff
@@ -0,0 +1,157 @@
+        url: 'https://wiki.metagame.wtf/docs/great-houses/house-of-daos',
+        icon: 'alliances',
+      },
+      {
+        title: 'Forum',
+        explainerText:
+          'Take part in slower & more thoughtful conversations on the forum',
+        url: 'https://forum.metagame.wtf/',
+        icon: 'forum',
+      },
+    ],
+  },
+  {
+    label: 'learn',
+    menuItems: [
+      {
+        title: 'MetaGame Wiki',
+        explainerText:
+          'New to MetaGame? This is the first thing you should dive deep into',
+        url: 'https://wiki.metagame.wtf/docs/wtf-is-metagame/wtf-is-metagame',
+        icon: 'metagamewiki',
+      },
+      {
+        title: 'Playbooks',
+        explainerText:
+          'Playbooks are short tutorials on achieving greatness, written by other players',
+        url: 'https://wiki.metagame.wtf/docs/playbooks/browse',
+        icon: 'playbooks',
+      },
+      {
+        title: 'Skill Trees',
+        explainerText:
+          'Skill Trees are meant to help people discover paths of self-development',
+        url: 'https://wiki.metagame.wtf/',
+        icon: 'learn',
+      },
+      {
+        title: 'Welcome to MetaGame',
+        explainerText:
+          'Just joined MetaGame? Check this for learning about how to get started',
+        url: 'https://wiki.metagame.wtf/',
+        icon: 'welcometometagame',
+      },
+      {
+        title: 'The Great Houses',
+        explainerText:
+          'The Great Houses are here to give people a meta view of different areas of interest',
+        url: 'https://wiki.metagame.wtf/docs/great-houses/house-of-daos',
+        icon: 'thegreathouses',
+      },
+      {
+        title: 'Asketh',
+        explainerText:
+          'Asketh is a place for players of MetaGame to ask whatever question they might have',
+        url: 'https://wiki.metagame.wtf/',
+        icon: 'asketh',
+      },
+    ],
+  },
+  {
+    label: 'contribute',
+    menuItems: [
+      {
+        title: 'Roles',
+        explainerText:
+          'Find about all roles in MetaGame, see which ones are open & how to play them',
+        url:
+          'https://wiki.metagame.wtf/docs/enter-metagame/roles-in-metagame#main-roles',
+        icon: 'roles',
+      },
+      {
+        title: 'Raids',
+        explainerText:
+          'See which raids are currently ongoing, learn more about them & get involved',
+        url: 'https://wiki.metagame.wtf/docs/enter-metagame/navigation-board',
+        icon: 'raids',
+      },
+      {
+        title: 'Quests',
+        explainerText:
+          'Take a look at available quest and claim ones that suit you best',
+        url: 'quests',
+        icon: 'quests',
+      },
+    ],
+  },
+  {
+    label: 'invest',
+    menuItems: [
+      {
+        title: 'Seed',
+        explainerText:
+          'Find out all about the Seeds; why, where & how you should buy them',
```

Comment:
```suggestion
          'Find out all about the SEEDs; why, where & how you should buy them',
```

I think the currency name is all caps, whether or not we're doing that here I don't know…